### ISSUE TITLE
Turning on climatisation doesn't respect climatisation settings

### ISSUE THREAD

**Issue by danielpervan** [CJNE/ha-porscheconnect#281]
## Version of the custom_component
0.1.2

## Describe the bug

Model: J1.2 MY2025 Taycan Cross Turismo

When climate control is activated through the integration, temperature appears to always be set to 20°C and all seat heating is turned off. However, when climate control is activated via the Porsche app, the last used temperature and seat heating settings are retained. If the user activates climate control through the integration and then opens the Porsche app, seat heating for all seats will appear as unchecked and the temperature setting is set to 20°C. This also affects scheduled departures, since they use the same settings for preconditioning.

Reproducible example:

1. In the Porsche app, turn on remote preconditioning with the temperature set to 21°C and driver seat heating turned on.
2. Turn off climatisation (Porsche app or integration).
3. In the Porsche app, tap on remote preconditioning and confirm that temperature is still set to 21° and driver seat heating is checked.
4. Turn on climatisation with the integration.
5. Turn off climatisation (Porsche app or integration).
6. In the Porsche app, tap on remote preconditioning.
7. Confirm in the Porsche app that temperature is now set to 20°C and all seat heating is unchecked.

Expected result: Temperature set to 21°C and driver seat heating is checked.

![IMG_0322](https://github.com/user-attachments/assets/32c92dc3-17e7-487c-8bc2-3ffa2029d39a)

**Comment by fredriklj** [comment]
Right, this is on my todo-list. Currently the integration calls the remote service with defaults in pyporscheconnectapi, which is 20 C and all seat heating off.

I'll try to fix this in the next release.

**Comment by Yura-87** [comment]
Awesome! This is exactly what I am looking for, especially now in winter season here in Europe :-) Thanks for your fantastic work @fredriklj 👏

**Comment by fredriklj** [comment]
Work in progress here: https://github.com/CJNE/ha-porscheconnect/tree/climatisation-service

Testers are welcome.

**Comment by Yura-87** [comment]
I will be happy to help with testing, but I don't know how to install this "branch" to HA?

**Comment by Dominik-Notzon** [comment]
@fredriklj Thanks for your quick work! I tested your change and I found two minor errors:

![image](https://github.com/user-attachments/assets/d3b46af5-6708-45ed-b889-0f758abdc370)

I can't fix them myself as I don't know how but the technical test passed on my side, I was able to use the service to start the climatisation at the desired temp, including the seat heating.

**Comment by fredriklj** [comment]
Thanks for spotting this! Fixed in 5ebefa7.
